tagilmo/utils/vereya_wrapper.py

```diff
@@ -252,13 +252,18 @@ def getChat(self, nAgent=0):
         else:
             return None
 
-    def getItemList(self, nAgent=0):
+    def getItemAndRecipeList(self, nAgent=0):
         self.sendCommand('item_list')
+        self.sendCommand('recipes')
         time.sleep(2)
```

Comment:
sleeps are not robust, you can build api on waitNotNoneObserve instead

Reply:
weird, but waitNotNoneObserver is currently returning None when i'm trying to implement getItemAndRecipeList using it.